### PR TITLE
EPMEDU-729 Create sandbox for UI components

### PR DESCRIPTION
**Pull request by igorTimofiejczyk**

### CODE REVIEW

**Review by diantyn — APPROVED** [review]

File: animealUI/Assemblers/TabBarControllerAssembler.swift [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import Foundation
+import UIKit
+import UIComponents
+import Style
+
+class TabBarControllerAssembler {
```
Comment (EugeniyGubar) [inline animealUI/Assemblers/TabBarControllerAssembler.swift]: Do we need `class` here? We could get by with `struct` or `enum`

File: animealUI/ComponentViewController.swift [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import UIComponents
+
+class ComponentViewController<T: UIView>: UIViewController {
+    typealias Element = T
+
+    // MARK: - Views
+    private var element: Element!
+
+    // MARK: - Private props
+    // private let dataSource: StylesDataSource
```
Comment (EugeniyGubar) [inline animealUI/ComponentViewController.swift]: commented code, will we need it in the future?

File: animealUI/ComponentViewController.swift [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    // MARK: - Public props
+    var configureElement: ((Element) -> Void)?
+
+    // MARK: - Lifecycle
+    override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
+        // dataSource = StylesDataSource()
```
Comment (EugeniyGubar) [inline animealUI/ComponentViewController.swift]: same here

File: animealUI/Assemblers/TabBarControllerAssembler.swift (hunk 1 above)
Comment (igorTimofiejczyk) [reply animealUI/Assemblers/TabBarControllerAssembler.swift]: changed to enum

File: animealUI/ComponentViewController.swift (hunk 2 above)
Comment (igorTimofiejczyk) [reply animealUI/ComponentViewController.swift]: done

File: animealUI/ComponentViewController.swift (hunk 3 above)
Comment (igorTimofiejczyk) [reply animealUI/ComponentViewController.swift]: done

**Review by EugeniyGubar — APPROVED** [review]

**Pull request merged**